resources/lang/en/rankings.php

```diff
@@ -23,6 +23,12 @@
         'multiplayer' => 'multiplayer',
         'performance' => 'performance',
         'score' => 'score',
+        'seasons' => 'seasons',
+    ],
+
+    'seasons' => [
+        'ongoing' => 'This season is still ongoing.',
```

Comment:
I wonder if the "ongoing" need further explanation (as in "there will be more playlists added") or it's already obvious to players. Because the other meaning of ongoing is there are still open playlists (similar to concluded, and also how I initially thought what they mean).

Reply:
The intent was to set concluded=1 after last playlist ends so pretty much "ongoing" would mean that.

Reply:
I mean, the season concludes when all added playlists are finished, so pretty much "ongoing" means the last added playlist is still open, and that possibly more playlists will be added (not a case for the very last playlist of the season). 

The chain is as it goes (with each step being exectued when the following playlist lobby closes):
Playlist A -> Playlist B -> Playlist C -> set concluded=1

Could change the label for something like "This season is still ongoing. (more playlists might be added)" or so?

Reply:
> season concludes when all added playlists are finished

yes I got that already after you explained it. But to me usually something is concluded when it's over, not when "the final round is open".

Similarly for ongoing, if a season is not ongoing anymore (the tooltip is gone), doesn't that mean it's over? (although in this case the date would be in the future so it's then clear that it's still... ongoing? open?)

Reply:
yes, the concluded is meant to be set to true when the season is over is what i mean.

Reply:
oh wait, I guess it's me who read it wrong 😵‍💫 I thought it'd be set to concluded when the final playlist is added
so there's no actual indication whether or not more playlists will be added? is it just going to be announced somewhere else or is it intentionally not public? and now I think about it, should the page contain some kind of description?

Reply:
The playlists are announced in spotlights newsposts and wiki article (e.g. https://osu.ppy.sh/home/news/2022-12-06-beatmap-spotlights-season-8-winter-2023-precision#season-schedule-and-leaderboards and https://osu.ppy.sh/wiki/en/Beatmap_Spotlights/Seasons/2023_Winter#schedule).

Hmm, guess some description could be used however I'm not sure if it's going to be used for spotlights only in the end.

Reply:
wouldn't it be more useful to have flag on whether or not more playlists will be added as I initially thought? that way the end date is visible right on the page when the final playlist is up without having to check the relevant page (and now I wrote it, at least an info link maybe?)

Reply:
i mean the room card already has this info so not really sure if that's needed, but we could go this route i guess (requires only column name / tooltip text cahnge i think 🤔)

 ![](https://i.imgur.com/Z4p2eCC.png)

Reply:
but it's not necessarily the end of the season (unless the room title says so)? also when the actual end is already known why not just show in the header

Reply:
yeah, changed concluded to finalised and added tooltip notice as suggested before